### DISCUSSION TITLE
Obisidian如何实现显示多个换行和空格?

### DISCUSSION THREAD

**Discussion by CocaColor** [obsidianzh/forum#57] [Q&A常见问题]
如下图, 当在编辑模式下有多个换行,将不会在预览模式下显示.

无论是直接`回车`, 还是`空格` + `空格` + `回车`, 都不起作用.
我现在的解决方法就是手动输入 `<br>` 换行符, 但是感觉很不方便.
![image](https://user-images.githubusercontent.com/79190419/108171850-11c20500-7137-11eb-9545-47d69095de53.png)
![image](https://user-images.githubusercontent.com/79190419/108172113-61083580-7137-11eb-94b0-70bcf9a400fe.png)

**Comment by musnows** [comment]
在设置里面吧“严格换行”关掉
